### PR TITLE
Fix YouTube example in Google Cast documentation

### PR DESCRIPTION
**Pull request by jrjefferies**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [ x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by fprokop** [comment]
Typo in the Alias

**Comment by jrjefferies** [comment]
Sorry - fixed

File: source/_integrations/cast.markdown [hunk 1]
```diff
@@ -332,19 +332,18 @@ Optional:
...
-        entity_id: media_player.my_chromecast
-      data:
-        media_content_type: cast
-        media_content_id: '
-          {
-            "app_name": "youtube",
-            "media_id": "dQw4w9WgXcQ"
-          }'
-      service: media_player.play_media
+alias: "Cast YouTube video to My Chromecast"
```
Comment (frenck) [inline source/_integrations/cast.markdown:335]: This feels out of line with all other examples. Does this mean all other examples are incorrect too? If so, why only adjust this one?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by jrjefferies** [comment]
Hi,  
The other ones worked correctly, but I was getting some of these types of errors on this youtube example
Invalid config for [script]: required key not provided @ data['sequence'][0]['domain']. Got None. (See /config/configuration.yaml, line 3).

I now think that is more my problem with how tweaked it to get it into the GUI script editor ( specifically the 'Edit in YAML') I obviously broke the yaml indenting.    

The formatting different was due to it being from the 'edit in YAML' output not the format used on the scripts.yaml file.
I have retested the existing example (direct from the web document) and I have them working.

Im going to withdraw the pull - sorry for using up some of your precious time

**Pull request closed without merging**